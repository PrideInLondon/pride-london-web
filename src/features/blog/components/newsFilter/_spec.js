import React from 'react'
import { shallow } from 'enzyme'
import NewsFilter from '.'

describe(NewsFilter.name, () => {
  it('renders with default values', () => {
    const filterType = {
      hexColour: '#000000',
      title: 'Mock Filter',
    }
    const wrapper = shallow(
      <NewsFilter filterType={filterType} handleClick={() => {}} />
    )
    expect(wrapper).toMatchSnapshot()
  })

  // TODO: Add more tests for different filter types?
})
