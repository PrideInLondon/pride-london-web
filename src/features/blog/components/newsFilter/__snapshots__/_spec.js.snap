// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsFilter renders with default values 1`] = `
<styles__FilterTextContainer
  backgroundColor="#000000"
  isButton={false}
  isOutline={false}
  onClick={[Function]}
>
  <styles__FilterText>
    Mock Filter
  </styles__FilterText>
</styles__FilterTextContainer>
`;
